packages/smooth_ui_library/lib/navigation/widgets/smooth_navigation_bar_classic.dart

```diff
@@ -84,9 +91,10 @@ class _SmoothNavigationBarClassicState extends State<SmoothNavigationBarClassic>
                                                   .textTheme
                                                   .bodyText1
                                                   .copyWith(
```

Comment:
I assume that `bodyText2` would be much better than `bodyText1.copyWith(...)`.

I'm not an expert on themes, but I think the point is to have homogenous colors and text sizes.
Of course for the moment it's a work in progress for all of us, but it would make sense to say: "OK guys, now we use themes, here are the available colors and text sizes, and that's it, no improvisation for each new widget."

What do you think of that?

Reply:
When designing the themes, I built in `bodyText2` as something like `onPrimary`, which should be used as the opposite of `bodyText1` in terms of colors. (Such as the icon on the scan button)
`bodyText2` currently has no use at all, that means we can use it for all kinds of purposes.
however, I think it is easier to use copyWith in this case, because it isn't a recurring text.

But yes, I agree with you we have to bring better structures into development. Such a kind of contribution guide that lists how to access the translations and how to add something to them, how to access the style elements and code guidelines.
Also, I think we need a better way of project planning, currently there are two roadmaps (README, Google Docs). Also, a big part of the documentation seems outdated.

@PrimaelQuemerais is currently working on a new one, when this is finished I'll look at it and attach a documentation of the data I have brought in and from what I is important.

But since this app is still in its early stage, we probably still have a bit time before we have to think about it seriously.

Reply:
The other suggestions you made are now in PR.